.github/workflows/on_push_action.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   ubuntu-build-and-test:
     runs-on: ubuntu-latest
     container:
-      image: mvcorreia/miplib-backends:1.0.1
+      image: gnosispm/miplib-backends:main
```

Comment:
I think the value here can't be "main" but should be whatever the current branch is. For example, I have a new change in the docker image, that is required for the current tests to pass, but it the CI is now using at the old docker image. Does this has a solution? @nlordell

Reply:
Since avoiding that is the purpose of this image altogether, I don't see any other solution than to hardcode the version there. Do you ?